Dockerfile.dapper

```diff
@@ -100,16 +96,16 @@ RUN cd /usr/src && \
     git clone https://github.com/rancher/liblonghorn.git && \
     cd liblonghorn && \
     git checkout 9eaf16c13dccc7f3bdb111d4e21662ae753bdccf && \
-    make deb && \
-    dpkg -i ./pkg/liblonghorn_*.deb
+    make && \
+    make install
 
 # Build TGT
 RUN cd /usr/src && \
     git clone https://github.com/rancher/tgt.git && \
     cd tgt && \
     git checkout e042fdd3616ca90619637b5826695a3de9b5dd8e && \
-    ./scripts/build-pkg.sh deb && \
-    dpkg -i ./pkg/tgt_*.deb
+    make && \
+    make install 
```

Comment:
The same as instance manager, do we still need these (tgt and liblonghorn build in the dapper image)? because you made the change to build it during the build time (package/Dockerfile).

Reply:
Yes, for the ci running in Drone.